frontend/src/pages/Policy/policyContent/FaqsContent.tsx

```diff
@@ -71,111 +74,53 @@ export const policyFaqs = [
-            </li>
-            <li>
-              <strong>Breakdowns by sex:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by urbanicity:</strong> No
-            </li>
-          </ul>
+          Currently, all of our gun violence datasets include national- and
+          state-level data.
         </p>
-
+        <DatasetList datasets={datasets} />
         <p>
-          <strong>Gun Suicides:</strong>
-          <ul>
-            <li>
-              <strong>Breakdowns by race/ethnicity:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by age:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by sex:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by urbanicity:</strong> No
-            </li>
-          </ul>
+          Additionally, (if the data is available) users can filter the
+          datasets' demographics by:
         </p>
+        <ul className='py-0 my-0'>
+          <li className='py-0 my-0'>
+            <strong>Age Groups:</strong> Available in 5-year increments and
+            single-year groups (ranging from 0 to 85+ years, including unknown
+            ages).
+          </li>
+          <li>
+            <strong>Sex:</strong> Data can be filtered for male and female
+            demographics.
+          </li>
+          <li>
+            <strong>Race and Ethnicity:</strong> Options include White, Black,
+            American Indian/Alaska Native, Asian, Hawaiian Native/Pacific
+            Islander, more than one race, unknown race, and Hispanic.
+          </li>
+        </ul>
       </>
     ),
   },
   {
     question: 'How is youth-related violence addressed in the data?',
     answer: (
       <>
-        Our dataset on youth-related gun violence specifically addresses gun
-        deaths among two age groups: children (0-17) and young adults (18-25).
-        For both age groups, the data is broken down by race and ethnicity,
-        providing insights into the racial and ethnic disparities that may exist
-        in gun-related deaths. However, the dataset does not currently include
-        other potential breakdowns, such as gender, socioeconomic status, or
-        geographic location.”
+        <p>
+          Our dataset on youth-related gun violence specifically addresses gun
+          deaths among two age groups: <strong>children (0-17)</strong> and{' '}
+          <strong>young adults (18-25)</strong>.
+        </p>
+        <p>
+          For both age groups, the data is broken down by race and ethnicity,
+          providing insights into the racial and ethnic disparities that may
+          exist in gun-related deaths.
+        </p>
+        <p>
+          However, the dataset does not currently include other potential
+          breakdowns, such as gender, socioeconomic status, or geographic
+          location.
```

Comment:
we shouldn't say "location" here since the data does have location by state; maybe "metro/non-metro status" or "urbanicity" or something?